app/controllers/lists_controller.rb

```diff
@@ -74,23 +74,22 @@ def update
 
       @tag.mark_as_dirty! if saved
 
-    @tag.mark_as_dirty! if saved
-
-    respond_to do |format|
-      format.html do
-        if saved
-          flash[:success] = "List updated"
-        else
-          flash[:danger] = "Could not save your list"
+      respond_to do |format|
```

Comment:
Just realised this commit should definitely be squashed into the previous one. I'll sort it once you've finished your review so it doesn't update the commit SHA's